README.md

```diff
@@ -44,19 +44,40 @@ cd physlr/src && make install PREFIX=/opt/physlr
 
 ## Generating Physlr Physical Map with stLFR reads
 
-To construct a physical map, you need linked reads from 10X Genomics or stLFR. In addition, to visualize the correctness and contiguity of the physical map, you need a reference genome.
-In this example, the linked reads and reference genome are called `linkedreads.fq.gz` and `reference.fa`, respectively. The linked reads are from stLFR so we specify `protocol=stlfr` to use the default value for stLFR reads.
+To construct a physical map _de novo_, you need linked reads (from 10X Genomics or MGI stLFR). 
 
+In this example, the linked reads dataset is called `linkedreads.fq.gz`. The linked reads are from stLFR so we specify `protocol=stlfr` to use the default value for stLFR reads.
+
+```
+cd experiment
```

Comment:
```suggestion
cd experiment # Change to working directory 
```
Just thinking to make it extra clear what you mean by this line?

Reply:
I agree, thank you for pointing out that things were unclear